File: natural_nums.py
def sum_nums(n):
    result = (n*(n+1))/2
    return result

def main():
    num = int(input("Enter the value of a number: "))
    natural_sum = int(sum_nums(num))
    print(f"The first total of the sum of natural numbers is {natural_sum}")

if __name__ == "__main__":
    main()

File: NaturalNums.java
import java.util.Scanner;

class NaturalNums {
    public static int sumNums(int num) {
        int totalNum = (num*(num+1))/2;
        return totalNum;
    }

    public static void main(String[] args) {
        int n;
        Scanner inp = new Scanner(System.in);
        System.out.print("Enter the value of n: ");
        n = inp.nextInt();
        int result = sumNums(n);
        System.out.printf("The sum of the first %d natural numbers is: %d\n", n, result);
        inp.close();
    }
}